src/index.js

```diff
@@ -1 +1,45 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+const username = document.getElementById('fullname');
+const userScore = document.getElementById('score');
+const btn = document.querySelector('form');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/UbOmK2M7Lmj63DwuaTqr/scores';
+
+const postScore = async () => {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+    body: JSON.stringify({
+      user: username.value,
+      score: userScore.value,
+    }),
+  });
+  btn.reset();
```

Comment:
Kindly check your POST method. The APP is not performing as expected. I am unable to POST score and if I click the refresh button, the APP is not responding. Kindly check the POST and refresh buttons.

Reply:
I don't understand because everything is working perfectly fine on my side